src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -35,11 +37,12 @@ class JsonAdaptedPerson {
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-            @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tags") List<JsonAdaptedTag> tags) {
+            @JsonProperty("email") String email, @JsonProperty("nus_net") String nusNet,
```

Comment:
Perhaps we should use `nusNet` instead of `nus_net`? According to the [Google Json Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format), property names must be camel-cased.
```suggestion
            @JsonProperty("email") String email, @JsonProperty("nusNet") String nusNet,
```

Reply:
My bad, I assumed that json is snaked_cased